sources/osgAnimation/RigTransformHardware.js

```diff
@@ -15,84 +19,74 @@ define( [
     function getShader( nbVec4 ) {
```

Comment:
well it was as first implementation to test without shader. If we dont need it anymore, we can remove it.